packages/documentation/src/content/docs/integration/prod/docker-compose.mdx

```diff
@@ -2,25 +2,53 @@
 title: Docker Compose
 ---
 
-This is an example deployment of Rafiki on a Linux virtual machine using nginx as a reverse proxy. You must change the values in the examples appropriate for your environment.
+import { LinkOut } from '@interledger/docs-design-system'
 
-## Virtual Machine preparation
+This guide is an example of deploying Rafiki using Docker Compose with Nginx as a reverse proxy on a virtual machine (VM) in a cloud environment. This guide also uses <LinkOut href='https://certbot.eff.org/'>Certbot</LinkOut> to generate <LinkOut href='https://letsencrypt.org/'>Let’s Encrypt TLS certificates</LinkOut> to secure exposed ports using HTTPS.
 
-Deploy the virutal machine:
+## Prerequisites
+
+### Domain and subdomains setup
+
+We will map the [Open Payments resource server](/integration/services/backend-service#open-payments) to your domain, and the [ILP Connector](/integration/services/backend-service#interledger-connector), [Open Payments auth server](/integration/services/auth-service), and [Admin UI](/integration/services/frontend-service) to subdomains. Using the DNS host of your choice, set up your domain and subdomains according to the following convention:
+
+| service                       | URL          | example            |
+| ----------------------------- | ------------ | ------------------ |
+| Open Payments resource server | DOMAIN       | myrafiki.com       |
+| ILP Connector                 | ilp.DOMAIN   | ilp.myrafiki.com   |
+| Open Payments auth server     | auth.DOMAIN  | auth.myrafiki.com  |
+| Admin UI                      | admin.DOMAIN | admin.myrafiki.com |
+
+:::note
+The example domain and subdomain values are for demonstration purposes only. You must use the actual domain names that you set up with your DNS host.  
+:::
+
+### VM specifications
+
+This example deployment uses the `e2-standard-2` VM on <LinkOut href='https://cloud.google.com/compute/docs/general-purpose-machines#e2_machine_types'>Google Cloud Platform</LinkOut> with the following specifications:
```

Comment:
Looking at this, maybe instead of specifying a specific cloud provider we can say something generic like, "a general-purpose VM with 4GB memory (I think 4GB should be enough actually) and 2 vCPUs should be capable of running the Rafiki software via docker."